src/preset.ts

```diff
@@ -868,7 +869,7 @@ export async function expandConfigurePreset(folder: string, name: string, worksp
                     shell: true
                 }).result;
 
```

Comment:
The `compilerLocation` variable above can be removed.

Reply:
In addition, now that we aren't going to be using `where.exe` to determine whether to include the VS environment, we can remove the `whereOutput` variables and corresponding code.

Reply:
I think we still need whereExecutable here right https://github.com/microsoft/vscode-cmake-tools/blob/6ea698014683af1a4984bb419574448bea8eb93f/src/preset.ts#L946

Reply:
We could do that, but we don't really need to. If the point of this change is to rely on the setting, then we shouldn't have any "magic". We should apply the vs dev environment if the setting is on, and don't if not.